src/Tribe/Views/V2/Widgets/Widget_Abstract.php

```diff
@@ -289,4 +332,67 @@ protected function args_to_context( array $arguments, Context $context ) {
 
 		return $context_args;
 	}
+
+	/**
+	 * Handles gathering the data for admin fields.
+	 *
+	 * @since TBD
+	 *
+	 * @param int                 $field_id    The ID of the field.
+	 * @param array<string,mixed> $field       The field info.
+	 * @param array<string,mixed> $passthrough Passthrough data (from a parent - like fieldset, to its children).
+	 * @param Context             $context     The Admin View current context.
+	 * @return void
```

Comment:
:no_entry_sign: Function return type is void, but function contains return statement

<sub>hash: cb98d58382179973be7db4886970eae5</sub>